.gitignore

```diff
@@ -12,3 +12,4 @@ _site
 dist
 glossary/glossary.yml
 man
+/Gemfile.lock
```

Comment:
why the leading `/`?

Reply:
that's what magit did... happy to remove it